addons/web/static/tests/search/custom_filter_item_tests.js

```diff
@@ -687,4 +687,37 @@ QUnit.module("Search", (hooks) => {
             ["id", "=", 9],
         ]);
     });
+
+    QUnit.test("delete button is visible", async function (assert) {
+        assert.expect(3);
```

Comment:
not necessary in this case (there's no assertions done in handlers that could or could not be executed)